README.md

```diff
@@ -168,34 +183,57 @@ sudo apt install libgl1 libpng16-16 libqt6core6 libqt6gui6 \
     iptables iputils-ping libatm1 libxtables12 xterm
 ```
 
-Then run multipass's daemon:
+Then run the Multipass daemon:
+
 ```
 sudo <multipass>/build/bin/multipassd &
 ```
 
-Copy the desktop file multipass clients expect to find in your home:
+Copy the desktop file that Multipass clients expect to find in your home:
 
 ```
 mkdir -p ~/.local/share/multipass/
 cp <multipass>/data/multipass.gui.autostart.desktop ~/.local/share/multipass/
 ```
 
-Optionally, enable auto-complete in bash:
+Optionally, enable auto-complete in Bash:
 
 ```
 source <multipass>/completions/bash/multipass
 ```
 
-Finally, use multipass's clients:
+Finally, launch the Multipass clients:
```

Comment:
Hmm, I'm not entirely sure this sentence fits. It makes sense to launch the GUI client, but not so much the CLI client. Maybe this could be expanded in two different sentences? Something like "Finally, you can use Multipass from the terminal using the CLI client \<insert example\>. Or you can launch the GUI \<insert GUI command\>". Or something along those lines :)

Reply:
Thanks @andrei-toterman , this makes sense! I'll rework this sentence :)

Reply:
Done: 

> Now you can use the `multipass` command from your terminal (for example `<multipass>/build/bin/multipass launch --name foo`) or launch the GUI client with the command `<multipass>/build/bin/multipass.gui`.